dev/AppNotifications/AppNotificationManager.cpp

```diff
@@ -57,72 +58,117 @@ namespace winrt::Microsoft::Windows::AppNotifications::implementation
 
     void AppNotificationManager::Register()
     {
-        winrt::guid storedComActivatorGuid{ GUID_NULL };
-        if (!PushNotificationHelpers::IsPackagedAppScenario())
+        HRESULT hr{ S_OK };
+        std::wstring appId{};
+
+        auto logTelemetry{ wil::scope_exit([&]() {
+            AppNotificationTelemetry::RegisterByAPI(hr, appId);
+        }) };
+
+        try
         {
-            std::wstring notificationId{ RetrieveNotificationAppId() };
-            if (!AppModel::Identity::IsPackagedProcess())
+            winrt::guid storedComActivatorGuid{ GUID_NULL };
+            if (!PushNotificationHelpers::IsPackagedAppScenario())
             {
-                THROW_IF_FAILED(PushNotifications_RegisterFullTrustApplication(notificationId.c_str(), GUID_NULL));
+                std::wstring notificationId{ RetrieveNotificationAppId() };
+                if (!AppModel::Identity::IsPackagedProcess())
+                {
+                    THROW_IF_FAILED(PushNotifications_RegisterFullTrustApplication(notificationId.c_str(), GUID_NULL));
 
-                storedComActivatorGuid = RegisterComActivatorGuidAndAssets();
-            }
+                    storedComActivatorGuid = RegisterComActivatorGuidAndAssets();
```

Comment:
Incorrect spacing\indenting everywhere here

Reply:
Please look into indenting in all the files and below as well.

Reply:
Indenting looks good to me now. Let me know if you find other issues

Reply:
Spacing should be fixed now

Reply:
Hmm not sure if a new iteration was sent with these changes but the indentation issues can still be seen above and below

Reply:
I think that it's because you're looking at the code through github. If you look at in in VS, then it's fine. Keep in mind that most the code was indented an extra tab when I introduced the try / catch